my-code/movie-search/src/Containers/MovieCatalogue/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+import { Link } from 'react-router-dom';
+
+import { fetchMovies } from '../../Actions';
```

Comment:
You're importing this action creator but it's never used

Reply:
Right, I wasn't able to take it out. Good catch!